src/utils/file_manager.h

```diff
@@ -0,0 +1,47 @@
+//
+// Created by blinku on 15.12.18.
+//
+
+#pragma once
+
+#include "pch.h"
```

Comment:
precompile header should be included in source file

Reply:
moved to src